File: multiples_of_3_and_5/python/main.py
# def multiple_sum(limit):
#     sum = 0
#     for i in range(limit):
#         if i % 3 == 0 or i % 5 == 0:
#             sum += i
#     return sum


def multiple_sum(limit):
    return sum([i for i in range(limit) if i % 3 == 0 or i % 5 == 0])


a = int(input("What limit? "))
print('Sum of all multiples of 3 and 5 from 0 to {}: {}'.format(a, multiple_sum(a)))

File: multiples_of_3_and_5/cpp/main.cpp
#include <iostream>
using namespace std;

int multiple_sum(int limit) {
  int sum = 0;
  for (int i = 0; i < limit; i++) {
    if (i % 3 == 0 or i % 5 == 0)
      sum += i;
  }
  return sum;
}

int main() {
  cout << "Sum of multiples of 3 and 5 from 0 to 10: " << multiple_sum(10)
       << endl;
  cout << "Sum of multiples of 3 and 5 from 0 to 1000: " << multiple_sum(1000)
       << endl;
}
